doc/user-guide/dask.rst

```diff
@@ -533,61 +436,50 @@ Notice that the 0-shaped sizes were not printed to screen. Since ``template`` ha
    As :py:func:`map_blocks` loads each block into memory, reduce as much as possible objects consumed by user functions.
    For example, drop useless variables before calling ``func`` with :py:func:`map_blocks`.
 
+Deploying Dask
+--------------
 
+By default, Dask distributes work across multiple cores on a single machine, which lets you process as much data as you can download and fit onto your hard drive. However, this has two limitations:
```

Comment:
I think the removed content about default `ThreadPoolExecutor` etc. should be re-added here.

Reply:
Oh good idea, I added it in.